keps/sig-storage/1710-selinux-relabeling/kep.yaml

```diff
@@ -36,6 +37,9 @@ feature-gates:
     components:
       - kube-apiserver
       - kubelet
+  - name: SELinuxChangePolicy
+    components:
+      - kube-apiserver
```

Comment:
nit: if we are adding a controller then, the flag should have to be enabled in KCM too right?

Reply:
good catch, added